packages/recorder/src/session_recording/session_recorder.ts

```diff
@@ -2,62 +2,96 @@ import fs from 'node:fs/promises';
 
 import {RecordedCall} from '../types';
 
+type HttpRequestSummary = {
+    host: string;
+    path: string;
+    method: string;
+    time: string;
+    filename: string;
+};
+
+type SessionSummary = {
```

Comment:
In terms of data flow of a program, what happens to a session throughout the lifecycle of one usage of full circle?

Reply:
I think I cleared this up in the readme linked in the PR description now. LMK if it makes more sense now